From 454512c02a47694a97dfa333c70dacfa034fd83c Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinh.nguyen@intel.com>
Date: Fri, 14 Feb 2020 13:48:08 -0600
Subject: [PATCH 066/151] HSD #14010901465-2: Backport Frame Buffer II driver
 from 4.14-ltsi

commit  d8c2ac87843aa7aeba380aae9a9e6d3639d2526f from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Because of commit("deb00d2785bed fbdev: make unregister/unlink
functions not fail"), the function unregister_framebuffer changed
from returning int to just a void.

Signed-off-by: Dinh Nguyen <dinh.nguyen@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 drivers/video/fbdev/altvipfb2.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/altvipfb2.c b/drivers/video/fbdev/altvipfb2.c
index e6d3b1cc6884..ef95a88ce2ea 100644
--- a/drivers/video/fbdev/altvipfb2.c
+++ b/drivers/video/fbdev/altvipfb2.c
@@ -178,7 +178,8 @@ int altvipfb2_remove(struct device *dev)
 			(void *)&fbpriv->info.screen_base,
 			fbpriv->info.fix.smem_start);
 
-	return unregister_framebuffer(&fbpriv->info);
+	unregister_framebuffer(&fbpriv->info);
+	return 0;
 }
 EXPORT_SYMBOL_GPL(altvipfb2_remove);
 
-- 
2.26.1

